packages/graphql/src/schema/create-relationship-fields/index.ts

```diff
@@ -571,162 +565,165 @@ function createRelationshipFields({
         const connectionUpdateInputName = `${rel.connectionPrefix}${upperFirst(rel.fieldName)}UpdateConnectionInput`;
         const relationshipWhereTypeInputName = `${sourceName}${upperFirst(rel.fieldName)}AggregateInput`;
 
-        const [nodeWhereAggregationInput, edgeWhereAggregationInput] = [n, relFields].map((nodeOrRelFields) => {
-            if (!nodeOrRelFields) {
-                return;
-            }
-
-            const fields = WHERE_AGGREGATION_TYPES.reduce<BaseField[]>((r, t) => {
-                const f = [...nodeOrRelFields.primitiveFields, ...nodeOrRelFields.temporalFields].filter(
-                    (y) => !y.typeMeta.array && y.typeMeta.name === t
-                );
-
-                if (!f.length) {
-                    return r;
+        if (rel.typeMeta.array) {
```

Comment:
Slightly nit-picky but could this not consolidated with the other  `if (rel.typeMeta.array)` statement that's a few lines below?

Reply:
I think I've resolved what you were after!